en/contributing/build.md

```diff
@@ -94,31 +91,24 @@ To build the *Dronecode SDK* C++ Library on Windows:
    ```sh
    git submodule update --init --recursive
    ```
-1. Download the [curl-7.56.1.zip](https://curl.haxx.se/download/curl-7.56.1.zip) source and extract it to the root of your C drive.
-1. Open the *VS2015 x64 Native Tools Command Prompt*, go to the source directory and enter:
-   ```sh
-   cd C:\curl-7.56.1\winbuild
-   nmake /f Makefile.vc mode=static VC=15 MACHINE=x64 DEBUG=no
-   ```
 1. Then build the SDK in Windows:
    ```sh
    cd /your/path/to/DronecodeSDK
-   mkdir build && cd build
-   cmake -DWIN_CURL_INCLUDE_DIR:STRING=C:\\curl-7.56.1\\include -DWIN_CURL_LIB:STRING="C:\curl-7.56.1\builds\libcurl-vc15-x64-release-static-ipv6-sspi-winssl\lib\libcurl_a.lib" -G "Visual Studio 15 2017 Win64" ..
-   cmake --build .
+   cmake -G "Visual Studio 15 2017" -DBUILD_SHARED_LIBS=ON -Bbuild/default -H.
+   cmake --build build/default --config Debug
    ```
 
-   > **Tip** You can generate *release* binaries by setting `--config Release` in the build step (`--config Debug` is used by default):
+   > **Tip** You can generate *release* binaries by setting `--config Release` in the build step:
      ```sh
-     cmake --build . --config Release
+     cmake --build build --config Release
      ```
    <span></span>
    > **Tip** To use more than one CPU core to build set this before building:
      ```sh
      set CL=/MP
      ```
 
-1. (Optionally) "Install" the SDK [as described below](#install-artifacts). This is required in order to build [Dronecode SDK C++ apps](../guide/toolchain.md), but not to run SDK test code.
```

Comment:
If this step is going to be optional then it is useful to explain why/when it is needed. So perhaps either add back a note like I had (or something better), or remove "Optionally".

Reply:
It's like any library, in the end. You can install it on your system if you want to have it on your system, or you can install it locally if you want to have it installed locally. There is no particular reason for one or the other. For instance I'm on Arch, and I don't install stuff manually on my system: I leave that to my package manager.

Maybe I should remove this bullet point completely and add information on how to use it (with a sample CMakeLists.txt for instance).

Reply:
I eventually went for removing the "Optionally". People who don't know what they are doing will be fine with installing, in the end :-).